app/src/main/res/values-fr/strings.xml

```diff
@@ -54,7 +54,7 @@
   <string name="btn_export">Exporter</string>
```

Comment:
Unfortunately, all changes in this file will be overriden after importing from CrowdIn, please do the transalations there